package.json

```diff
@@ -30,15 +30,14 @@
     "test": "npm run lint && npm run spec"
   },
   "engines": {
-    "node": ">=6.0.0"
+    "node": ">= 8.0.0"
   },
   "dependencies": {
     "debug": "^4.1.1",
-    "electron-installer-common": "^0.6.2",
+    "electron-installer-common": "electron-userland/electron-installer-common#drop-node-6",
```

Comment:
`-debian` should be fine. `-redhat` will need to be fixed, and I have a WIP branch locally for that.

Reply:
I just run a test on `-debian` with the master branch of `-common` and it shows an error on the lintian test, which is the only error I was afraid might happen.
Because we don't default to `revision=1`, we need to change this line https://github.com/electron-userland/electron-installer-debian/blob/drop-node-6/test/installer.js#L150 to `changelog-file-missing-in-native-package`